DucksNet.API.Integration Tests/Usings.cs

```diff
@@ -0,0 +1,2 @@
+﻿global using Xunit;
+global using FluentAssertions;
```

Comment:
What is this?

Reply:
e generata de Visual Studio, ca sa nu scrii in fiecare unit test, scrii intr-un fisier si este global

Reply:
aha... personal nu prea imi place dar e ok. :shipit: